docs/contributing/setup.md

```diff
@@ -44,7 +44,7 @@ To test with Nextflow for development purpose:
 
 ## Change and preview the docs
 
-The docs are generated using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/). To change the docs, edit the files in the [docs/](docs/) folder and run the following command to generate the docs (after installing mkdocs via `pip install mkdocs-material`):
+The docs are generated using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/). To change the docs, edit the files in the [docs/](https://github.com/nextflow-io/nf-validation/tree/master/docs) folder and run the following command to generate the docs (after installing mkdocs via `pip install mkdocs-material`):
```

Comment:
not so sure about this change: they should have it locally when editing, so I would keep a local link here.

Reply:
The problem was that it was trying to link to here: https://nextflow-io.github.io/nf-validation/contributing/setup/docs/ which doesn't exist

Reply:
ah, true. how about `[docs/](../)` then instead?

Reply:
true, I was thinking wrong on how the links are resolved on the website, nevermind.

Reply:
I agree with @nvnieuwk 👍🏻  These docs will be read on the web, not the local clone. So should point to the repo.